CHANGELOG.md

```diff
@@ -3,6 +3,83 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.13.0
+
+_Aug 15, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Allow [label editing capabilities on the Tree View](https://mui.com/x/react-tree-view/rich-tree-view/editing/) (#13388) @noraleonte
+- 🔧 Improve rows accessibility on the Data Grid features "Tree Data" and "Row Grouping". See the [documentation](https://mui.com/x/react-data-grid/row-grouping/#accessibility-changes-in-v8) for more information. (#13623) @arminmeh
```

Comment:
```suggestion
- 🔧 Improve rows accessibility on the Data Grid features "Tree Data" and "Row Grouping". See the [documentation](https://mui.com/x/react-data-grid/row-grouping/#accessibility-changes-in-v8) for more information.
```

Reply:
```suggestion
- 🔧 Improve rows accessibility on the Data Grid features "Tree Data" and "Row Grouping". Certain "Row Grouping" accessibility updates will only be applied if experimental feature flag is enabled. See the [documentation](https://mui.com/x/react-data-grid/row-grouping/#accessibility-changes-in-v8) for more information.
```

Reply:
I took @arminmeh suggestion because it adds an additional explanation ;)